apps/wallet-dashboard/app/(protected)/migrations/page.tsx

```diff
@@ -16,62 +20,74 @@ import {
     Panel,
     Title,
 } from '@iota/apps-ui-kit';
+import { Assets, Clock, IotaLogoMark, Tokens } from '@iota/ui-icons';
 import { useCurrentAccount, useIotaClient } from '@iota/dapp-kit';
 import { STARDUST_BASIC_OUTPUT_TYPE, STARDUST_NFT_OUTPUT_TYPE, useFormatCoin } from '@iota/core';
-import { useGetStardustMigratableObjects } from '@/hooks';
-import { useQueryClient } from '@tanstack/react-query';
-import { Assets, Clock, IotaLogoMark, Tokens } from '@iota/ui-icons';
 import { IOTA_TYPE_ARG } from '@iota/iota-sdk/utils';
+import { StardustObjectMigrationType } from '@/lib/enums';
+import { MigrationObjectsPanel } from '@/components';
 
 interface MigrationDisplayCard {
     title: string;
     subtitle: string;
     icon: React.FC;
 }
 
-function MigrationDashboardPage(): JSX.Element {
+export default function MigrationDashboardPage(): JSX.Element {
```

Comment:
revert this to keep consistence with the rest of the pages :pray: 
```suggestion
function MigrationDashboardPage(): JSX.Element {
```

Reply:
This shouldn't be like that, and should be refactored later, but ok

Reply:
wdym?

Reply:
we should export all components directly from the function, without a default export. So, for default exports like `page.tsx|layout.tsx` it will make more sense to just add `export default function Page() {` because  all other components will be `export function Component() {`
This way no component will have an extra export line at the end